js/content/store.js

```diff
@@ -1385,10 +1385,24 @@ class AppPageClass extends StorePageClass {
         if (this.isDlc()) { return; }
 
         let appid = this.appid;
-        let response = await Background.action("appdetails", {"appids": appid, "filters": "support_info"});
-        if (!response || !response[appid] || !response[appid].success) { return; }
+        let cache = LocalStorage.get("support_infos", {});
```

Comment:
Why `LocalStorage` instead of `CacheStorage`?

Reply:
I though that too, but if you'd use `CacheStorage` it would expire the whole `support_infos` even though they were fetched at different times

Reply:
You can cache per-appid. It would be almost automatic if we refactored `endpointFactoryCached` so it was available on `SteamStore`.

Reply:
It will be easy to do once IndexedDB support is added

Reply:
So, we can do this another time perhaps.